src/components/LongTermAverageGraph/LongTermAverageGraph.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {values, keys} from 'lodash';
+
+
+//this weird sequence avoids crashing npm when plotly is loaded.
+import Plotly from 'plotly.js';
+import createPlotlyComponent from 'react-plotly.js/factory';
+const Plot = createPlotlyComponent(Plotly);
+
+
+function LongTermAverageGraph({longTermData}) {
+
+    /* data is a temporary variable that represents the data, which are
+     key value pairs. The keys are the date of the measurement, the values
+     are the temperature at that date. The keys will be sorted, and then the
+     temperatures will be matched to the correct years. */
+    let data = longTermData.rXi1p1.data;   
```

Comment:
Unfortunately, we can't rely on the name of the run being `rXi1p1`. Sometimes it will be `r1i1p1` or `r2i1p1` instead. Each time a simulation is run with one of the climate models is run, it gets a different run code like this. Sometimes we have data from several different simulations available.

Instead of assuming the name of the run will always be `r1i1p1`, could you rewrite this so that _no matter what the name of the run is_, its data is loaded?

Sorry I wasn't clear about this requirement - I'm so used to our data, I forget how odd some of it is!

Reply:
Actually, could you add the simulation code to the title of the graph as well?